apps/ensadmin/src/components/recent-registrations/utils.tsx

```diff
@@ -51,3 +52,21 @@ export function Duration({
 
   return <>{duration}</>;
 }
+
+/**
+ * An integer value (representing a Unix timestamp in seconds) formatted as a string.
+ */
+export type UnixTimestampInSeconds = string;
+
+/**
+ * Transforms a timestamp formatted as a string to a Date object.
```

Comment:
```suggestion
 * Transforms a UnixTimestampInSeconds to a Date.
```

Best to build upon definitions you've already invested in.